pkgs/servers/sql/postgresql/ext/promscale_extension.nix

```
@@ -0,0 +1,60 @@
+{ lib
+, fetchFromGitHub
+, fetchpatch
+, buildPgxExtension
+, postgresql
+, stdenv
+, nixosTests
+}:
+
+buildPgxExtension rec {
+  inherit postgresql;
+
+  pname = "promscale_extension";
+  version = "0.8.0";
+
+  src = fetchFromGitHub {
+    owner = "timescale";
+    repo = "promscale_extension";
+    rev = version;
+    sha256 = "sha256-vyEfQMGguHrHYdBEEmbev29L2uCa/4xL9DpGIniUwfI=";
+  };
+
+  cargoSha256 = "sha256-vq/R9Kd0a9ckgEYm9Kt2J+RdyiHHYVutNEM4dcRViUo=";
+
+  cargoPatches = [
+    # there is a duplicate definition in the lock file which fails to build with buildRustPackage
+    (fetchpatch {
+      name = "cargo-vendor.patch";
+      url = "https://patch-diff.githubusercontent.com/raw/timescale/promscale_extension/pull/611.patch";
```

```suggestion
      url = "https://github.com/timescale/promscale_extension/commit/3cef3f26f72ebf52d8800910ea655cac09312c57.patch";
```
we want to use an url which does not change if more things are pushed to the PR